keyboards/dztech/dz65rgb/keymaps/jumper149/readme.md

```diff
@@ -0,0 +1,21 @@
+# jumper149's DZ65RGB v2
+
+Run commands in the root directory of this repository.
+
+```
+./bin/qmk compile && sudo dfu-programmer atmega32u4 erase && sudo dfu-programmer atmega32u4 flash ./dztech_dz65rgb_v2_jumper149.hex && sudo dfu-programmer atmega32u4 reset
+```
+
+## build
+
+```
+./bin/qmk compile
```

Comment:
You can use `qmk flash` to flash.

Reply:
I would like to leave the comments on the manual flashing process there, as I had some trouble with it.